packages/composites/gm-token/src/transport/price.ts

```diff
@@ -95,9 +96,9 @@ export class GmTokenTransport extends SubscriptionTransport<GmTokenTransportType
       sources,
     } = await this.fetchPrices(assets, providerDataRequestedUnixMs)
 
-    const indexToken = tokenAddresses.arbitrum[index]
-    const longToken = tokenAddresses.arbitrum[long]
-    const shortToken = tokenAddresses.arbitrum[short]
+    const indexToken = tokenAddresses.arbitrum[index as keyof typeof tokenAddresses.arbitrum]
+    const longToken = tokenAddresses.arbitrum[long as keyof typeof tokenAddresses.arbitrum]
+    const shortToken = tokenAddresses.arbitrum[short as keyof typeof tokenAddresses.arbitrum]
```

Comment:
```suggestion
    const indexToken = tokenAddresses.arbitrum[index.toUpperCase() as keyof typeof tokenAddresses.arbitrum]
    const longToken = tokenAddresses.arbitrum[long.toUpperCase() as keyof typeof tokenAddresses.arbitrum]
    const shortToken = tokenAddresses.arbitrum[short.toUpperCase() as keyof typeof tokenAddresses.arbitrum]
```

Reply:
Actually noticed that `WBTC.b` isn't uppercase, so this doesn't make sense. Closing!